### PR TITLE
Run upstart as UPSTART_UID if set

### PR DESCRIPTION
**Pull request by timbrandin**
Something like this? Just typed it up, not tested..

### CODE REVIEW

File: templates/linux/meteor.conf [hunk 1]
```diff
@@ -15,13 +15,21 @@ script
...
 
     ##add custom enviromntal variables
     if [ -f config/env.sh ]; then
       . config/env.sh
     fi
-
-    ##start the app
-    forever -c userdown --minUptime 2000 --spinSleepTime 1000 app/main.js
+    
+    if [ -lt $PORT 1024 ]
```
Comment (arunoda) [inline templates/linux/meteor.conf]: If UPSTART_UID is not set, we need to go for userdown as normally.

**Comment by timbrandin** [comment]
What do you think @arunoda, this could work right?

File: templates/linux/meteor.conf [hunk 2]
```diff
@@ -20,8 +20,13 @@ script
     if [ -f config/env.sh ]; then
       . config/env.sh
     fi
-
-    ##start the app
-    forever -c userdown --minUptime 2000 --spinSleepTime 1000 app/main.js
+    
+    if [ -z $UPSTART_UID ]; then
```
Comment (arunoda) [inline templates/linux/meteor.conf:24]: Yeah! this seems like correct for me.

**Comment by arunoda** [comment]
Did you test this? I mean deployed?

I'll also test this in weekend.
In the meantime, try to create a wiki entry on this and link it to [additional resources](https://github.com/arunoda/meteor-up#additional-resources) section.

**Comment by timbrandin** [comment]
I haven't tested it no, just the bash scripts on my dev server. I have no clue how to try it out locally.

I'm not sure where to place documentation on this, but I guess somewhere after multiple deployments?

**Comment by timbrandin** [comment]
Started working on some documention... I'm not sure if this information is what you wanted me to add?
https://github.com/arunoda/meteor-up/wiki/Using-Meteor-Up-with-NginX-vhosts

**Comment by arunoda** [comment]
I did test this and merged.
Released a new version: 0.7.0

**Comment by timbrandin** [comment]
Great! :D

**Pull request merged**